please help me address this error:
```
homeassistant/components/github/sensor.py:145:12: W0201: Attribute '_self_latest_release_tag' defined outside __init__ (attribute-defined-outside-init)
```

--- a/homeassistant/components/github/sensor.py
+++ b/homeassistant/components/github/sensor.py
@@ -123,9 +123,10 @@ class GitHubSensor(Entity):
             ATTR_OPEN_PULL_REQUESTS: self._pull_request_count,
             ATTR_STARGAZERS: self._stargazers,
         }
-        if self._latest_release_tag is not None:
-            attrs[ATTR_LATEST_RELEASE_TAG] = self._latest_release_tag
-        return attrs
+        if self._github_data.latest_release_url is not None:
+            self._latest_release_tag = self._github_data.latest_release_url.split("tag/")[1]
+        else:
+            self._latest_release_tag = None
 
     @property
     def icon(self):

--- a/homeassistant/components/github/sensor.py
+++ b/homeassistant/components/github/sensor.py
@@ -123,10 +123,9 @@ class GitHubSensor(Entity):
             ATTR_OPEN_PULL_REQUESTS: self._pull_request_count,
             ATTR_STARGAZERS: self._stargazers,
         }
-        if self._github_data.latest_release_url is not None:
-            self._latest_release_tag = self._github_data.latest_release_url.split("tag/")[1]
-        else:
-            self._latest_release_tag = None
+        if self._latest_release_tag is not None:
+            attrs[ATTR_LATEST_RELEASE_TAG] = self_latest_release_tag
+        return attrs
 
     @property
     def icon(self):
@@ -142,8 +141,8 @@ class GitHubSensor(Entity):
         self._available = self._github_data.available
         self._latest_commit_message = self._github_data.latest_commit_message
         self._latest_commit_sha = self._github_data.latest_commit_sha
-        if self._latest_release_url is not None:
-            self._latest_release_tag = self._latest_release_url.split("tag/")[1]
+        if self._github_data.latest_release_url is not None:
+            self._latest_release_tag = self._github_data.latest_release_url.split("tag/")[1]
         else:
             self._latest_release_tag = None
         self._latest_release_url = self._github_data.latest_release_url

--- a/homeassistant/components/github/sensor.py
+++ b/homeassistant/components/github/sensor.py
@@ -111,12 +111,11 @@ class GitHubSensor(Entity):
     @property
     def device_state_attributes(self):
         """Return the state attributes."""
-        return {
+        attrs = {
             ATTR_PATH: self._repository_path,
             ATTR_NAME: self._name,
             ATTR_LATEST_COMMIT_MESSAGE: self._latest_commit_message,
             ATTR_LATEST_COMMIT_SHA: self._latest_commit_sha,
-            ATTR_LATEST_RELEASE_TAG: self._latest_release_tag,
             ATTR_LATEST_RELEASE_URL: self._latest_release_url,
             ATTR_LATEST_OPEN_ISSUE_URL: self._latest_open_issue_url,
             ATTR_OPEN_ISSUES: self._open_issue_count,
@@ -124,6 +123,9 @@ class GitHubSensor(Entity):
             ATTR_OPEN_PULL_REQUESTS: self._pull_request_count,
             ATTR_STARGAZERS: self._stargazers,
         }
+        if self._latest_release_tag is not None:
+            attrs[ATTR_LATEST_RELEASE_TAG] = self._latest_release_tag
+        return attrs
 
     @property
     def icon(self):
@@ -140,9 +142,9 @@ class GitHubSensor(Entity):
         self._latest_commit_message = self._github_data.latest_commit_message
         self._latest_commit_sha = self._github_data.latest_commit_sha
         if self._latest_release_url is not None:
-            self._self_latest_release_tag = self._latest_release_url.split("tag/")[1]
+            self._latest_release_tag = self._latest_release_url.split("tag/")[1]
         else:
-            self._self_latest_release_tag = None
+            self._latest_release_tag = None
         self._latest_release_url = self._github_data.latest_release_url
         self._state = self._github_data.latest_commit_sha[0:7]
         self._open_issue_count = self._github_data.open_issue_count
@@ -181,7 +183,6 @@ class GitHubData:
         self.available = False
         self.latest_commit_message = None
         self.latest_commit_sha = None
-        self.latest_release_tag = None
         self.latest_release_url = None
         self.open_issue_count = None
         self.latest_open_issue_url = None

